### PR TITLE
hw2_kdh

### PR DESCRIPTION
**Pull request by DH117117**

### CODE REVIEW

File: class02/homework/kangdonghyuck/README.md
Comment (JerryJLee) [inline class02/homework/kangdonghyuck/README.md:1]: 불필요한 파일은 커밋에 포함시키지 않도록 합시다.

File: class02/homework/kangdonghyuck/hw02/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+./splitted_dataset/val/<good>: 6
+```
+
+## Training 결과
+|Classification model|Accuracy|FPS|Training time|Batch size|Learning rate|Other hyper-prams|
+|----|----|----|----|----|----|----|
+|EfficientNet-V2-S| |
+|EfficientNet-B0| 
+|DeiT-Tiny|
+|MobileNet-V3-large-1x|1.0 |8||0:00:06.579616||64|0.0058|-|
```
Comment (JerryJLee) [inline class02/homework/kangdonghyuck/hw02/README.md:22]: 3개 모델에 대해선 실패 했나요?
